modules/windows-workload-management.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * windows_container_workloads/understanding-windows-container-workloads.adoc
+
+[id="windows-workload-management_{context}"]
+= Windows workload management
+
+To run Windows workloads in your cluster, you must first install the Windows Machine Config Operator (WMCO). The WMCO is a Linux-based Operator that runs on Linux-based control plane and compute nodes. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+
+.WMCO design
+image::wmco-design.png[WMCO workflow]
+
+Before deploying Windows workloads, you must create a Windows compute node and have it join the cluster. The Windows node hosts the Windows workloads in a cluster, and can run alongside other Linux-based compute nodes. You can create a Windows compute node by creating a Windows machine set to host Windows Server compute machines. You must apply a Windows-specific label to the machine set that specifies a Windows OS image that has the Docker container runtime add-on enabled.
```

Comment:
s/Docker container/Docker-formatted container per the corporate guidelines

Reply:
@codyhoag I'd drop in a note somewhere in the docs & mention the [Docker deprecation](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) and mention that Windows will be moving to containerD once its available. @aravindhp pls LGTM below statements.

Currently we use Docker as the container runtime in Windows nodes, but containerD will be supported from Kubernetes 1.20 and that will be the recommendation going forth.

Reply:
I would rephrase it to `Currently we use Docker as the container runtime in Windows nodes, but will move to containerD once it goes GA`. or something along those lines. I am not 100% sure if ContainerD will hit GA in 1.20.

Reply:
@aravindhp @Anandnatraj thanks for the suggestion. I definitely agree on keeping the version GA as a general "future release of Kubernetes." I will add this note where we reference Docker so we're transparent on our future intentions.

Reply:
I have added `Docker-formatted container` language for this requirement and added the Docker deprecation/containerd note in each instance. This has been approved by PM.